_drafts/2015-07-06-web-audio-article-1.md

```diff
@@ -0,0 +1,106 @@
+
+###Let's get started!
+
+- #####Making sound in the browser! 
+
+	What is an **AudioContext**?
+
+	It is an interface that represents an audio-processing graph built from audio 	modules linked together, each represented by an **AudioNode**. These 	**AudioNodes** defines how the audio stream flows from its source (eg. an audio 	file) to its destination (eg. your speakers).
+
+	Remember you can use [Jsfiddle](http://jsfiddle.net/) to test out these codes!
+
+	Before we are able to do anything, we must first create an **AudioContext**
+
+	`var audioContext = new AudioContext()`
+
+	A single audio context can support multiple sound inputs and complex audio 	graphs.
+
+	Inside the **AudioContext** you are able to connect **AudioNodes**. There are 	different types of **AudioNodes**:
+
+	- Source nodes
+
+	- Modification nodes
+
+	- Analysis nodes
+
+	- Destination nodes
+
+	This line of code below creates an [OscillatorNode](https://developer.mozilla.org/en-US/docs/Web/API/OscillatorNode).
+
+	`var oscillator = audioContext.createOscillator()`
+
+	It is a type of **AudioNode** 	which is 	categorised under source nodes. Other 	than *oscillators*, source 	nodes includes 	other sound sources such as *audio 	buffers*, *live audio inputs*, `<audio>` tags and *JS 	processors*. You can read more about the other nodes [here](https://developer.mozilla.org/en-US/docs/Web/API/Web_Audio_API).
+
+
+	After you've created the sound source, it's time to connect it to your speakers! 
+	
+	`oscillator.connect(audioContext.destination)`
+
+	**audioContext.destination** is a special node that is associated with the 	default audio output of your system. It is categorised under destination 	nodes.
+
+	Here are we setting the properties of the [OscillatorNode](https://developer.mozilla.org/en-US/docs/Web/API/OscillatorNode).
+
+	`oscillator.type = 'sine'`
+
+	`oscillator.frequency.value = 100`
+	
+	Most properties on Audio Nodes are instances of [AudioParam](https://	developer.mozilla.org/en-US/docs/Web/API/AudioParam).
+	
+	Now to start your oscillator!
+
+	`oscillator.start(audioContext.currentTime)`
+
+	`oscillator.stop(audioContext.currentTime + 5)` // stop after 5 seconds
+
+	More info on the [OscillatorNode start method](https://developer.mozilla.org/en-	US/docs/Web/API/OscillatorNode/start) and [OscillatorNode stop method](https://	developer.mozilla.org/en-US/docs/Web/API/OscillatorNode/stop).
+
+	By the end of it your code should look like [this](http://jsfiddle.net/tsuLof5c/). 
```

Comment:
You can actually embed the entire JSFiddle in this page. 

http://doc.jsfiddle.net/use/embedding.html